src/lib/wallet/GoodWalletClass.js

```diff
@@ -1331,6 +1331,11 @@ export class GoodWallet {
     })
     return res
   }
+
+  async isKnownFuseAddress(address) {
+    const nonce = await this.wallet.eth.getTransactionCount(address)
```

Comment:
is there a faster way ? e.g. to find first transaction matching the address, without counting them all

Reply:
this is not counting. it requests from the blockchain the last "nonce" of that address. which is basically the transaction count.